### DISCUSSION TITLE
GitHub Support Not Responding to Ticket

### DISCUSSION THREAD

**Discussion by DelibasicIzet** [community/community#113681] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My ticket has been open for about 15 days now, and I haven't received a response from GitHub support. My primary account has been flagged, and I can't access support through any other means. Can someone help me? Is there another way to contact support to have my account unflagged?

**Comment by muhtasimwm** [comment]
hey @DelibasicIzet! hope you're doing well, could you please tell me what might be the reason your account got flagged by github?

**Reply by DelibasicIzet** [reply]
The thing is, I don't know why my account is flagged. We had a programming competition and everything was working fine; the project is on my profile, so I was just updating my personal CV document. Somehow, when I placed it in a folder, it got marked as some dangerous file, and since then, my account has been flagged, at least I assume. But I deleted that PDF and PNG file from my readme repository. I don't see any other reason why they would flag me, even though those documents were there before, there was nothing suspicious about them. Anyway, I don't know the real reason. Support hasn't responded to my ticket for 15 days.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @DelibasicIzet, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
